frontend/model/settings/vuexModule.js

```diff
@@ -70,17 +70,25 @@ const mutations = {
     state.increasedContrast = isChecked
   },
   setNotificationEnabled (state, enabled) {
-    if (state.notificationEnabled !== enabled) {
-      // We do this call to `service-worker` here to avoid DRY violations.
-      // The intent is creating a subscription if none exists and letting the
-      // server know of the subscription
-      sbp('service-worker/setup-push-subscription').catch(e => {
-        // The parent `if` branch should prevent infinite loops
-        sbp('state/vuex/commit', 'setNotificationEnabled', false)
-        console.error('[setNotificationEnabled] Error calling service-worker/setup-push-subscription', e)
-      })
-    }
+    console.info('[setNotificationEnabled] set to:', enabled)
     state.notificationEnabled = enabled
+    // if necessary, prevents the service working from ignoring our notificationEnabled
+    // setting (which is not stored in the SW because it's Vuex-only) from requesting push
+    // notifications upon reconnecting to server
+    sbp('chelonia/configure', { disableNotifications: !enabled }).catch(e => {
```

Comment:
1. I don't think this belong in Chelonia configuration, especially since Chelonia has nothing to do with handling these settings.
2. This isn't a persisted setting, so this will not work. This needs to be a persisted setting, so probably it needs to be saved in the identity contract preferences.

Reply:
@corrideat I don't see why it matters that it's not persistent. It's set on startup. Are you saying the service worker loses all values in memory as it goes through its lifecycle? If so, that is a problem, but also tying Chelonia to a specific contract doesn't sound ideal either.

Either way, we can tackle this in a separate issue, not in this PR. Can you open that issue?

Reply:
Yes, the SW loses all values in memory.

> If so, that is a problem, but also tying Chelonia to a specific contract doesn't sound ideal either.

I don't think so, because none of this is done by Chelonia.

> we can tackle this in a separate issue, not in this PR

It's this PR that breaks it

Reply:
> I don't think so, because none of this is done by Chelonia.

Fair enough.

> It's this PR that breaks it

What breaks what?

Reply:
Users can 'opt out' of receiving notifications w/o resetting permissions, but this will not be honoured and will leave the app in an inconsistent state. I'm about to test exactly what this means for the app as a whole.

Reply:
I've now tested this and it's broken in this way:

1. Grant permission
2. Notifications are sent normally
3. Disable notifications using the app (permission intact)
4. Notifications stop
5. SW wakes up for some reason (which doesn't send the `LOGIN_COMPLETE`, i.e., without a browser client, such as in response to a periodic event, user manually waking it up, etc.)
6. Notifications start being sent again
7. Open app
8. Notification setting shows notifications off
9. Notifications are still being sent